src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,17 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: 
+      transform $transform-duration $linear-transform,
+      color $transform-duration $linear-transform;
+
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
```

Comment:
I think, we will you this catalog for next topics and this parameter could be used again, so can be variables